import java.util.Scanner;
class TriangularPark {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        // Enter side1, side2 and side3 in meters
        double side1 = input.nextDouble();
        double side2 = input.nextDouble();
        double side3 = input.nextDouble();

        double perimeter = side1 + side2 + side3;
        double rounds = 5000 / perimeter;

        System.out.println("The total number of rounds the athlete will run is "+ rounds +" to complete 5 km");
		
		//Closing Scanner obj
		input.close();
    }
}

